ntoskrnl/se/token.c

```diff
@@ -1844,6 +1860,43 @@ SepCreateToken(
         goto Quit;
     }
 
+/* Fill in token debug information */
+#if DBG
+    /*
+     * We must determine ourselves that the current
+     * process is not the initial CPU one. The initial
+     * process is not a "real" process, that is, the
+     * Process Manager has not yet been initialized and
+     * as a matter of fact we are creating a token before
+     * any process gets created by Ps. If it turns out
+     * that the current process is the initial CPU process
+     * where token creation execution takes place, don't
+     * do anything.
+     */
+    if (PsGetCurrentProcess() != &KiInitialProcess)
+    {
+        RtlCopyMemory(AccessToken->ImageFileName,
+                      PsGetCurrentProcess()->ImageFileName,
+                      min(sizeof(AccessToken->ImageFileName), sizeof(PsGetCurrentProcess()->ImageFileName)));
+
+        /*
+         * Set up the process and thread CIDs.
+         *
+         * !!!IMPORTANT NOTE!!! -- ProcessCid, ThreadCid and CreateMethod
+         * are member field names taken from Windows Server 2003 SP2 checked
+         * build kernel symbols. The reason why we have to have to such names
+         * is for WinDBG debug extension commands support purposes (such as
+         * !logonsession command respectively). THESE FIELD NAMES MUST NOT BE
+         * CHANGED!!!
+         */
```

Comment:
It makes more sense to have a comment like this in the header where it's defined?